### ISSUE TITLE
How to train the model available on PyTorch Hub?

### ISSUE THREAD

**Issue by augustoolucas** [ultralytics/yolov5#1535]
I would like to train the model available on PyTorch Hub, is it possible? Is there an available loss function to use?
I read the source code but I find the [training process](https://github.com/ultralytics/yolov5/blob/3b57cb56412976847b91ef184ca2f2a99491d8ab/train.py#L36) and the [loss computation](https://github.com/ultralytics/yolov5/blob/3b57cb56412976847b91ef184ca2f2a99491d8ab/utils/general.py#L478) too complicated to reproduce. 

Is there a simpler way to do the training? I've searched for some tutorials but didn't find anything.

**Comment by glenn-jocher** [comment]
@augustoolucas there's a very simple way to train:
```
python train.py
```

**Comment by ronly2460** [comment]
I also would like to know

**Comment by glenn-jocher** [comment]
@ronlynes see **Train Custom Data** tutorial:

## YOLOv5 Tutorials

* [Train Custom Data](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data)&nbsp; 🚀 RECOMMENDED
* [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/)&nbsp; ☘️ RECOMMENDED
* [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)&nbsp; 🌟 NEW
* [Supervisely Ecosystem](https://docs.ultralytics.com/yolov5/tutorials/model_export8)&nbsp; 🌟 NEW
* [Multi-GPU Training](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training)
* [PyTorch Hub](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading)&nbsp; ⭐ NEW
* [TorchScript, ONNX, CoreML Export](https://docs.ultralytics.com/yolov5/tutorials/model_export) 🚀
* [Test-Time Augmentation (TTA)](https://docs.ultralytics.com/yolov5/tutorials/test_time_augmentation)
* [Model Ensembling](https://docs.ultralytics.com/yolov5/tutorials/model_ensembling)
* [Model Pruning/Sparsity](https://docs.ultralytics.com/yolov5/tutorials/model_pruning_and_sparsity)
* [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution)
* [Transfer Learning with Frozen Layers](https://docs.ultralytics.com/yolov5/tutorials/transfer_learning_with_frozen_layers)&nbsp; ⭐ NEW
* [TensorRT Deployment](https://github.com/wang-xinyu/tensorrtx)
